PagingSample/app/src/main/java/paging/android/example/com/pagingsample/CheeseViewModel.kt

```diff
@@ -16,25 +16,23 @@
 
 package paging.android.example.com.pagingsample
 
-import android.app.Application
 import androidx.lifecycle.AndroidViewModel
-import androidx.paging.Pager
-import androidx.paging.PagingConfig
-import androidx.paging.PagingData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import androidx.paging.*
```

Comment:
do we normally use * imports in samples?

Reply:
Hmmm good point - I think the default code styles just does this, but I just updated my settings locally to fix this.